package.json

```diff
@@ -20,6 +20,7 @@
     "html-react-parser": "^5.1.10",
     "mapbox-gl": "^3.3.0",
     "react": "^18.2.0",
+    "react-bootstrap": "^2.10.2",
```

Comment:
It doesn't look like we're actually using any of the components from this library, so we can probably remove this.

Reply:
It was removed.